gherkin/elixir/lib/gherkin/errors.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Gherkin.ErrorMessage do
```

Comment:
It's pretty confusing that these are all in the same file.

Reply:
The implementations are kept as close as possible across languages so that a bug fix or improvement in one language can easily propagate. I considered moving them into their own directories, but opted to maintain the structure used by the other languages.